src/appimagetool.c

```diff
@@ -156,12 +156,13 @@ int sfs_mksquashfs(char *source, char *destination, int offset) {
             args[i++] = "16384";
         } else if (strcmp(sqfs_comp, "zstd") == 0) {
             /*
-             * > Build with 1MiB block size
-             * > Using a bigger block size than mksquashfs's default improves read speed and can produce smaller AppImages as well
-             * -- https://github.com/probonopd/go-appimage/commit/c4a112e32e8c2c02d1d388c8fa45a9222a529af3
+             * > Build with default 128K block size
+             * > It used to be 1M but that actually causes much higher startup times.
+             * > Some testing might be needed to see if there is some other value that actually improves performance.
+             * -- https://github.com/AppImage/appimagetool/issues/64
              */
             args[i++] = "-b";
-            args[i++] = "1M";
+            args[i++] = "128K";
```

Comment:
Do we need to define this _at all_? Default to me would mean "don't specify it".

Reply:
Explicit is better than implicit, isn't it?

Reply:
Well the goal is to use the default value. Why not leave that choice to the tool?

The value should work for zsync2, though.